templates/vertical-full-page-map/page.html.hbs

```diff
@@ -17,7 +17,7 @@
     {{> templates/vertical-full-page-map/script/locationbias modifier="main" }}
     {{> templates/vertical-full-page-map/script/locationbias modifier="mobileMap" }}
   {{/script/core }}
-  <script id="js-answersLocatorBundleScript" src="{{relativePath}}/locator-bundle.js"
+  <script id="js-answersLocatorBundleScript" src="{{relativePath}}/VerticalFullPageMap.js"
     onload="window.locatorBundleLoaded = true;" defer></script>
```

Comment:
nit: now window.locatorBundleLoaded is a little out of sync with the other naming. Maybe we should rename to `verticalFullPageMapBundleLoaded`?

Reply:
yes definitely! renamed other usages of LocatorBundle too